recipe/meta.yaml

```diff
@@ -1,16 +1,19 @@
 {% set name = "jupyter_server" %}
-{% set version = "1.4.1" %}
+{% set version = "1.13.5" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b0126f237f679533eec46244cfc66d61e3a1f8ec0fad2d47352fa19d3e754e47
+  sha256: 9e3e9717eea3bffab8cfb2ff330011be6c8bbd9cdae5b71cef169fcece2f19d3
 
 build:
   number: 0
+  noarch: python
+  # anyio >=3.1.0,<4 and websocket-client currentrly aren't available on s390x.
+  skip: True  # [s390x]
```

Comment:
Making it noarch above, means it will be available (though probably not installable due to dependencies) from the noarch pocket also on s390x, are we ok with that?

Reply:
I can remove skip s390x, but then the build process fails on s390x.

Reply:
Done